sweeper.lua

```diff
@@ -0,0 +1,45 @@
+local S = moreblocks.intllib
+local check_protection = moreblocks.check_protection
+local sweepables = {}
+
+-- example sweepable node
+-- TODO API for sweepable node registration
+sweepables["default:mossycobble"] = function(pos, node)
+	-- hit 3 times to clean
+	if node.param1 < 2 then
+		node.param1 = node.param1 + 1
```

Comment:
`param1` is usually used to store light information, so this might create issues when light sources get involved or when using this on nodes with different paramtype.

You want to store this kind of information in the [node metadata table](https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L1775).